frontend/src/errors/AuthErrors.ts

```diff
@@ -0,0 +1,29 @@
+import { PresentableError } from "../types/ErrorTypes";
+
+export const authErrors: Record<string, PresentableError> = {
```

Comment:
If we store all errors in one map, what are the odds we get duplicate messages? Or will we only ever use 'axiosError' thus we know there is a unique set of messages.

Reply:
`authErrors` ends up being our own custom error "dictionary" - we define what kinds of errors from the backend we want to accept & give special presentation treatment to. On the backend side we'll have to decide which error code to give, and then on the frontend we just look it up in the "dictionary" we've defined here - so there shouldn't be duplicates since we're defining the entirety of the object